### DISCUSSION TITLE
Can it compute the dynamic polarisability of HFS?

### DISCUSSION THREAD

**Discussion by balabalabalalaba** [nikolasibalic/ARC-Alkali-Rydberg-Calculator#125] [Q&A] [answered]
In the doc, it can compute the dynamic polarisability of FS. How can I compute the dynamic polarisability of HFS?

**Comment by nikolasibalic** [comment, marked as answer]
Currently not supported, but you can use existing calculations to extend functionality. If you do so, please consider opening a pull request to add to main repo!
